stdlib/3/builtins.pyi

```diff
@@ -682,28 +682,28 @@ class set(MutableSet[_T], Generic[_T]):
     def __contains__(self, o: object) -> bool: ...
     def __iter__(self) -> Iterator[_T]: ...
     def __str__(self) -> str: ...
-    def __and__(self, s: AbstractSet[Any]) -> Set[_T]: ...
-    def __iand__(self, s: AbstractSet[Any]) -> Set[_T]: ...
+    def __and__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __iand__(self, s: AbstractSet[object]) -> Set[_T]: ...
     def __or__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
     def __ior__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
-    def __sub__(self, s: AbstractSet[Any]) -> Set[_T]: ...
-    def __isub__(self, s: AbstractSet[Any]) -> Set[_T]: ...
+    def __sub__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __isub__(self, s: AbstractSet[object]) -> Set[_T]: ...
     def __xor__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
     def __ixor__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
-    def __le__(self, s: AbstractSet[Any]) -> bool: ...
-    def __lt__(self, s: AbstractSet[Any]) -> bool: ...
-    def __ge__(self, s: AbstractSet[Any]) -> bool: ...
-    def __gt__(self, s: AbstractSet[Any]) -> bool: ...
+    def __le__(self, s: AbstractSet[object]) -> bool: ...
+    def __lt__(self, s: AbstractSet[object]) -> bool: ...
+    def __ge__(self, s: AbstractSet[object]) -> bool: ...
+    def __gt__(self, s: AbstractSet[object]) -> bool: ...
     # TODO more set operations
 
 class frozenset(AbstractSet[_T], Generic[_T]):
     def __init__(self, iterable: Iterable[_T] = ...) -> None: ...
     def copy(self) -> FrozenSet[_T]: ...
-    def difference(self, *s: Iterable[Any]) -> FrozenSet[_T]: ...
-    def intersection(self, *s: Iterable[Any]) -> FrozenSet[_T]: ...
+    def difference(self, *s: Iterable[object]) -> FrozenSet[_T]: ...
```

Comment:
Hmm, good catch both of these should be the same. 

However, I think we should change the `__sub__` function, in order to allow the following code:
```py
from typing import Set, Union, FrozenSet

s1 = frozenset([1, 2])  # type: FrozenSet[int]
s2 = frozenset([1, '2'])  # type: FrozenSet[Union[str, int]]

# currently, the following line gives you an error but I don't think it should.
s3 = s1 - s2  # type: FrozenSet[int]
```

What do you think?